src/Methods/Modular_Programming/Hard/Q1/RightPrime.java

```diff
@@ -0,0 +1,42 @@
+package Methods.Modular_Programming.Hard.Q1;
+import java.util.Scanner;
+
+/*
+Write a program that will ask the user for a number and determine if it is a "right-truncatable prime",
+which is a special type of number where after removing each digit from the right, the resulting number
```

Comment:
This question text was updated on the Junits examples document, but it does not look like the changes made it to the main document, and probably not onto the website.

Reply:
I just double-checked the Junit examples doc and the text looks the same to me. What are you seeing that's different?

Reply:
@Hedgemon4 Do you still think it's different?

Reply:
Yeah, this is the same text in the examples document, but not the same as the gamification document

Reply:
Main doucment says: 
[J] Write a program that will ask the user for a number and determine if it is a "right-truncatable prime", which is a special type of number where after removing each digit from the right, the resulting number after each digit removal is a prime number.
Eg: 2393 (prime)
239 ( Remove 3, still prime)
23 (remove 9, still prime)
2 (remove 3, still prime)
Make sure to use at least one java method in your solution.

Reply:
So you're saying I should update it in the gamification document? We've been making changes to the questions in the examples document to make them better questions for a while now, the current version I have is likely to be the correct one.